### ISSUE TITLE
Blank squares on side bar

### ISSUE THREAD

**Issue by fcastilloec** [linuxmint/nemo#1586]
```
 * Nemo version: 3.4.7
 * Is issue with desktop or windowed nemo? windowed nemo
 * Distribution - Ubuntu 16.04.3
 * nVidia GTX-1070 using nvidia drivers v384.59 from ppa:graphics-drivers/ppa
 * 64 bit
 ```

**Issue**
The sidebar is displaying blank sections on the top or bottom. Depending on where the scroll bar is, this can be made more obvious when scrolling.
This GIF shows the blank section, both at the top and/or bottom, depending on the scrolling. I've also added a GIF that shows when you try to keep scrolling past the end.
![nemo_error_2](https://user-images.githubusercontent.com/1280530/29477705-8581dba0-8437-11e7-871c-e1081ebb5963.gif)
![nemo_error](https://user-images.githubusercontent.com/1280530/29477720-97bf3574-8437-11e7-87c6-e02d8f34a242.gif)
On previous versions, there was no blank square.

**Steps to reproduce**
It should be visible immediately, but scrolling makes it more obvious
Here's a GIF of how it looks when scrolling.

**Expected behaviour**
The blank areas shouldn't be there

**Other information**
I think something similar is happening with other themes, as reported in issue #1582 but that issue was closed since the blame was passed to the arc theme instead of nemo itself. The default Ambiance theme hasn't been updated, only nemo, so nemo is the one that broke it.

**Comment by fcastilloec** [comment]
Closing the issue without any explanation, or a comment or anything???????
Can somebody care to explain what's going on?

**Comment by leigh123linux** [comment]
We don't care about issues caused by use outside of cinnamon!

**Comment by fcastilloec** [comment]
Oh! I see... I guess you have a point.... Sorry about that. I should have known before posting it

**Comment by leigh123linux** [comment]
It actually looks like the theme has broken overshoot scroll

https://zoltanp.github.io/2017/06/gtk3-scroll-fix.html

**Comment by feren** [comment]
Yeah, I was going to mention that that usually means the GTK has a missing overshoot image...

**Comment by brownsr** [comment]
If this is using one of our themes then please say which one.

**Comment by feren** [comment]
@brownsr That looks like stock Ubuntu Ambiance...

**Comment by leigh123linux** [comment]
This what the overshoot is meant to look (arc-theme dark).
![vokoscreen-2017-08-19_12-55-17](https://user-images.githubusercontent.com/1299302/29486475-36d28b6a-84de-11e7-890a-c37d5c44c4b8.gif)

**Comment by fcastilloec** [comment]
@leigh123linux your link looked very promising but after adding the sections to Ambience, nemo is still behaving the same way. I have logged-out and back it. Restart nemo, but nothing.
So, I decided to try a different theme, something that will actually have an overshoot and undershoot defined, and I still get the same. Here's one using the same arc-theme dark (which does have those definitions) 
![peek 2017-08-20 15-39](https://user-images.githubusercontent.com/1280530/29497848-e3fec8be-85bd-11e7-897c-d121faca9cf6.gif)
